### PR TITLE
create teams playoffs table - closes #114

### LINKED ISSUE #114: Create visual identification on the table for teams in the playoffs and play-in
#### Description
I as a user, would like to have a visual identification on the table of which team is qualifying for the playoffs or play-in.

### PR DESCRIPTION
**Pull request by fsilvaco**
## Description
This PR solves #114  

## Before
Captura de Tela 2022-03-21 às 22 43 29

## Now
Captura de Tela 2022-03-21 às 22 48 38

Captura de Tela 2022-03-21 às 22 45 31

### CODE REVIEW

File: app/components/StandingTable/index.tsx [hunk 1]
```diff
@@ -28,9 +28,21 @@ export function StandingTable({ label, conference }: StandingTableProps) {
...
-              <TableCell>{team.rank}</TableCell>
+              <TableCell>
+                <div
+                  className={` rounded-full ${
+                    index < 6
+                      ? 'bg-green-900'
+                      : index < 10
+                      ? 'bg-green-600'
+                      : ''
+                  }`}
```
Comment (willianjusten) [inline app/components/StandingTable/index.tsx:41]: What do you think about extracting this to a variable and use it here?

**Comment by willianjusten** [comment]
It looks good to me, I'll just ping @adeonir because he's our "designer" guy xD

File: app/components/StandingTable/index.tsx (hunk 1 above)
Comment (fsilvaco) [reply app/components/StandingTable/index.tsx:41]: Does it only refer to the names of the color classes? If yes, what do you think about being in the `constants.ts` file?

Comment (willianjusten) [reply app/components/StandingTable/index.tsx:41]: No no, I meant doing something like https://github.com/willianjusten/nba-remix/blob/main/app/components/Footer/index.tsx#L2

You create a variable at the top with the logic and then just reuse it here.

Comment (adeonir) [reply app/components/StandingTable/index.tsx:41]: I think that this could have unrelated colors:

image

Comment (adeonir) [inline app/components/StandingTable/index.tsx:41]: You can use the `classnames` package here:

```suggestion
                  className={cn('rounded-full', {
                    'bg-green-600': index <= 5,
                    'bg-sky-600': index >= 6 && index < 10,
                  })}
```

File: app/components/StandingTable/index.tsx [hunk 2]
```diff
@@ -49,6 +61,16 @@ export function StandingTable({ label, conference }: StandingTableProps) {
           ))}
         </tbody>
       </Table>
+      <div className="flex justify-center">
+        <div className="mr-6 flex items-center">
+          <div className="mr-2 h-5 w-10 rounded-full bg-green-900"></div>
```
Comment (adeonir) [inline app/components/StandingTable/index.tsx:66]:
```suggestion
          <div className="mr-2 h-5 w-10 rounded-full bg-green-600"></div>
```

File: app/components/StandingTable/index.tsx [hunk 3]
```diff
@@ -49,6 +61,16 @@ export function StandingTable({ label, conference }: StandingTableProps) {
...
+          <span className="text-sm text-gray-400">Playoffs</span>
+        </div>
+        <div className="flex items-center">
+          <div className="mr-2 h-5 w-10 rounded-full bg-green-600"></div>
```
Comment (adeonir) [inline app/components/StandingTable/index.tsx:70]:
```suggestion
          <div className="mr-2 h-5 w-10 rounded-full bg-sky-600"></div>
```

File: app/components/StandingTable/index.tsx (hunk 1 above)
Comment (helderburato) [reply app/components/StandingTable/index.tsx:41]: I agree with @adeonir, it seems better with the approach he mentioned above.

Comment (helderburato) [reply app/components/StandingTable/index.tsx:41]: What do you think about extracting this logic to give meaningful names to the logic, i.e:
```js
className={cn('rounded-full', {  
  'bg-green-600': isTopSixTeams(),
  'bg-sky-600': IsTopTenTeams(),
})}
```

Something like that.

**Comment by fsilvaco** [comment]
Hi guys! Thanks for the comments! I made a commit with the suggestions. 🚀

**Review by willianjusten — APPROVED** [review]
Looks great =)

File: app/utils/handleQualification/index.ts [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export const isPlayoff = (i: number) => i <= 5
+export const isPlayin = (i: number) => i >= 6 && i < 10
```
Comment (adeonir) [inline app/utils/handleQualification/index.ts:2]: Just to keep the convention from other functions:

```suggestion
export function isPlayoff(i: number) {
  return i <= 5
}

export function isPlayin(i: number) {
  return i >= 6 && i < 10
}
```

Comment (willianjusten) [reply app/utils/handleQualification/index.ts:2]: I believe he took from my suggestion, so it's my fault if I wrote it different than our conventions hahaha

**Pull request merged**